src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
It's better to implement Drawable and Area interfaces here so that each class that extends Figure also implements Drawable and Area by default